circle.yml

```diff
@@ -31,7 +31,8 @@ dependencies:
 test:
   override:
     - gulp check
-    - gulp test
+    - gulp karma
+    - gulp isotest
```

Comment:
i just added `test-dist` task yesterday so either add that here or revert this change back to `gulp test`